Makefile.global

```diff
@@ -62,12 +62,11 @@ endef
 # Library objects required in multiple places:
 LIBSBASE   = $(addprefix $(TOPDIR)/lib/,lib.error lib.misc)
 LIBHEADERS = $(addsuffix .h,$(LIBSBASE))
-LIBSOURCES = $(addsuffix .c,$(LIBSBASE))
 LIBOBJECTS = $(addsuffix $(OBJEXT),$(LIBSBASE))
 CFLAGS   += -I$(TOPDIR)/lib -I$(TOPDIR)/etc
 CXXFLAGS += -I$(TOPDIR)/lib -I$(TOPDIR)/etc
 
-$(LIBOBJECTS): %$(OBJEXT): %.c %.h
+$(LIBOBJECTS):
```

Comment:
Did you test that these changes work and trigger a rebuild if you change either the source or the header? I'm assuming you're relying on implicit make rules?